experiments/evaluate_config_generator.py

```diff
@@ -12,20 +12,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from experiments.experiment_evaluator import ExperimentEvaluator
```

Comment:
Fix the title of this PR. The changes in this PR are unrelated to that title.

Reply:
Fixed the title.

Reply:
I changed it again (I think the title is what gets put as the commit message, so I want it to be accurate)